fars/api/views.py

```diff
@@ -16,14 +16,19 @@ class Meta:
         model = Booking
         fields = ['bookable', 'before', 'after']
 
-
 class BookingsList(viewsets.ViewSetMixin, generics.ListAPIView):
-    permission_classes = [IsAuthenticated]
-    queryset = Booking.objects.all()
+
     serializer_class = NoMetaBookingSerializer # Exclude metadata to hide doorcode in this API
     filter_backends = (filters.DjangoFilterBackend,)
     filter_class = BookingFilter
 
+    def get_queryset(self):
+        queryset = Booking.objects.all()
+        if not self.request.user.is_authenticated:
+            queryset = queryset.filter(bookable__public=True)
+        
+        return queryset
```

Comment:
Note to reviewer: the idea here is that if the user is not logged in, they can still see bookings for public bookables. Hence we remove the permission_classes but add a forced filter if the user is not authenticated.